openedx/core/djangoapps/auth_exchange/views.py

```diff
@@ -86,12 +96,6 @@ class DOTAccessTokenExchangeView(AccessTokenExchangeBase, DOTAccessTokenView):
 
     oauth2_adapter = adapters.DOTAdapter()
 
-    def get(self, request, _backend):
```

Comment:
[verifying] This is being removed because we removed tho super call that would result in this getting executed, is that right?

Reply:
I basically had moved this to be the super version, but I'm going to remove it altogether I think, and just go with `allowed_methods` as detailed elsewhere.